src/gsad_gmp.c

```diff
@@ -10276,6 +10320,380 @@ get_system_report_gmp_from_url (gvm_connection_t *connection,
   return NULL;
 }
 
+/**
+ * @brief Get one report config, envelope the result.
+ *
+ * @param[in]  connection     Connection to manager.
+ * @param[in]  credentials    Username and password for authentication.
+ * @param[in]  params         Request parameters.
+ * @param[in]  extra_xml      Extra XML to insert inside page element.
+ * @param[out] response_data  Extra data return for the HTTP response.
+ *
+ * @return Enveloped XML object.
+ */
+static char *
+get_report_config (gvm_connection_t *connection, credentials_t *credentials,
+                   params_t *params, const char *extra_xml,
+                   cmd_response_data_t *response_data)
+{
+  gmp_arguments_t *arguments;
+  arguments = gmp_arguments_new ();
+
+  gmp_arguments_add (arguments, "alerts", "1");
```

Comment:
If that is more or less required shouldn't we just remove this option and make it the default on gvmd side?

Reply:
I've modeled this after the get_report_formats command but I've removed it for simplicity now.